src/diff/changes/apiChanges.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { NodeChanges } from "./nodeChanges";
+import { RuleCategory } from "../ruleSet";
+
+/**
+ * Holds changes between two API specifications
+ */
+export class ApiChanges {
+  //Changes between base and new API specifications
+  public nodeChanges: NodeChanges[];
+
+  /**
+   * Create ApiChanges object
+   * @param baseApiSpec - Base API specification to compare
+   * @param newApiSpec - New API specification to compare
+   */
+  constructor(public baseApiSpec: string, public newApiSpec: string) {
+    this.nodeChanges = [];
```

Comment:
This was previously passed as an argument. Why was that changed?

Reply:
it does have to be a constructor argument. Changes can be set after the object is created